server/views/pages/componentRequestForm.njk

```diff
@@ -12,8 +12,8 @@
   <div class="govuk-grid-column-one-half">
     {% include 'partials/errorSummary.njk' %}
     <h1 class="govuk-heading-xl">Component Request Form</h1>
+    <p>Documentation for creating new services is located <a class="govuk-link--no-visited-state" href="https://tech-docs.hmpps.service.justice.gov.uk/creating-new-services/">here</a>. If this is the first time you're creating a service please make sure you read that first. 
     <p>After submitting this form, the SRE team will run the bootstrap to create the github repo and service catalogue entries. </p>
-    <p><i><b>Note:</b> fields marked '*' are required.</i></p>
```

Comment:
swapped over to follow gov uk guidance around not using asterisk and using `(optional)` instead